gravitee-exchange-connector/gravitee-exchange-connector-websocket/src/main/java/io/gravitee/exchange/connector/websocket/client/WebSocketEndpoint.java

```diff
@@ -76,6 +71,6 @@ public String getHost() {
     }
 
     public String resolvePath(String path) {
-        return uri.resolve(path).getRawPath();
+        return url.getPath() + (path.startsWith("/") ? path : "/" + path);
```

Comment:
if url.getPath() ends with a / and path doesn't not, you will come up with //

Reply:
That's why I've remove any trailing / in the `newEndpoint` method.
Tests are covering all the case I think (tell me if you think of another case)